manifests/AdoptOpenJDK/OpenJDK/11.0.7.yaml

```diff
@@ -0,0 +1,14 @@
+Id: AdoptOpenJDK.OpenJDK
+Version: 11.0.7
+Name: AdoptOpenJDK 11.0.7+10 (x64)
+Publisher: AdoptOpenJDK
+Homepage: https://adoptopenjdk.net
+License: GPL 2 with Classpath Exception
+LicenseUrl: https://github.com/AdoptOpenJDK/openjdk-jdk/blob/master/LICENSE
+Description: Community-driven, Vendor-neutral build of OpenJDK for Windows x64.
+Installers: 
+    - Arch: x64
+      Url: https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.7%2B10.2/OpenJDK11U-jdk_x64_windows_hotspot_11.0.7_10.msi
```

Comment:
I'd recommend using the Adopt API over the direct connection to GitHub which is rate limited

Reply:
How do a get an MSI installer with the API?

https://api.adoptopenjdk.net/swagger-ui/#/Binary/get_v3_binary_version__release_name___os___arch___image_type___jvm_impl___heap_size___vendor_

Reply:
Sadly [Adopt API](https://api.adoptopenjdk.net/swagger-ui/#/Binary) doesnt provide an endpoint for the installer (msi) and only for the binary (zip). 

Latest jdk 11 binary (.zip) could be retreived with following api call:
https://api.adoptopenjdk.net/v3/binary/version/jdk-11.0.7%2B10.2/windows/x64/jdk/hotspot/normal/adoptopenjdk?project=jdk

And winget doesnt seem to support installs of .zip yet, see https://github.com/microsoft/winget-pkgs/issues/174

So I dont know if the author of this pr can solve the situation.